packages/shared/src/lib/timezones.ts

```diff
@@ -519,3 +519,19 @@ export const getTimeZoneOptions = (): { value: string; label: string }[] => {
     return { ...timeZone, label: timeZoneCurrentOffsetLabel(timeZone) };
   });
 };
+
+export const getUserDefaultTimezone = (): string => {
+  const timeZoneOptions = getTimeZoneOptions();
+  const timeZoneOffset = (new Date().getTimezoneOffset() / 60) * -1;
+  const timezoneGuess = timeZoneOptions.find((timezone) =>
+    timezone.label.startsWith(
+      `(UTC ${timeZoneOffset > 0 ? '+' : ''}${timeZoneOffset})`,
+    ),
+  );
+
+  if (timezoneGuess) {
+    return timezoneGuess.value;
+  }
+
+  return 'Europe/London';
+};
```

Comment:
Is there any more accurate way? Because there could be multiple timezones with the same offset

Reply:
The difficulty lies within how JS timezones compare to browser timezones.
So IANA timezones are by far way more than actual timezones used in JS.

We could extend this function to map manually based on all IANA timezones?

See for reference: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

A browser would for instance return:
Africa/Johannesburg 

Where my timezone is:  Africa/Windhoek

Even moment does guesstimating on the timezone.
But the user would be "close" to it's offset.

Reply:
Ok so let's keep it as it is. Thanks for the info